pkg/frontend/nvme_controller.go

```diff
@@ -76,12 +79,16 @@ func (s *Server) CreateNvmeController(_ context.Context, in *pb.CreateNvmeContro
 		return nil, err
 	}
 
+	ctrlrID := autoCtrlrIDAllocation
+	if in.NvmeController.Spec.NvmeControllerId != nil {
+		ctrlrID = int(*in.NvmeController.Spec.NvmeControllerId)
+	}
```

Comment:
maybe this needs to be in the https://github.com/opiproject/opi-marvell-bridge/commit/799d7b34e5acecda48c25c48bf775b9c4024725f commit ?

Reply:
In this case, it will be better to put in a separate commit then: f04093df21190eba68623e62280e61033ff6344f